package.json

```diff
@@ -33,7 +33,8 @@
     "sense": "nebula sense",
     "spec": "scriptappy-from-jsdoc -c ./spec-configs/props.conf.js",
     "test:unit": "aw -c aw.config.js",
-    "test:integration": "aw puppet --testExt '*.int.js' --glob 'test/integration/**/*.int.js'"
+    "test:integration": "aw puppet --testExt '*.int.js' --glob 'test/integration/**/*.int.js'",
+    "extension": "yarn build && cp -r ./dist ~/Qlik/Sense/Extensions"
```

Comment:
Don't want to push this I think, it's very specific to developing against sense-client

Reply:
@haxxmaxx  I imagine that we will develop against sense-client quite often. Do you think so?

Reply:
We will, but we should keep the references to private code to a minimum. i don't think this command  is applicable if you don't have access to sense-client

Reply:
Good point. Did not consider that. I will remove it.